jsx_file.jsx

```diff
@@ -0,0 +1,6 @@
+
+var a = 2
+
+function(beer) {
```

Comment:
_[farcy v0.1b](https://github.com/appfolio/farcy)_

* Missing name in function declaration.